tests/constants.py

```diff
@@ -37,3 +37,17 @@
 
 # path of the file created automatically by
 JSON_CSV_PATH = os.path.join(TEST_REPO_PATH, INPUT_FOLDER, CSV_ELEMENTS, CSV_FNAME)
+
+# input directory for tests
+TEST_INPUT_DIRECTORY = "inputs"
+# input directory for file you need for specific tests
+SPECIFIC_TEST_INPUT_DIRECTORY = "inputs_for_specific_tests"
```

Comment:
I see that F1 has already created 2 "specific" test input folders, in a sense the input folders are only specific to D1, so I would write it explicitely within the variable name, or define this variable name directly in the `tests/test_D1_model_components.py` module.

We could also put every needed input folder within the `testes/test_data` folder so that the `tests` folder does not become to crowded with input folders?

Reply:
> We could also put every needed input folder within the `testes/test_data` folder so that the `tests` folder does not become to crowded with input folders?

+1 

I already had the feeling that there was some cleaning up to do...

Reply:
I'll do it then :)